product_supplierinfo_qty_multiplier/models/purchase_order_line.py

```diff
@@ -4,47 +4,41 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 from math import ceil
 
-from openerp.tools import frozendict
-
 from odoo import _, api, models
 
 
 class PurchaseOrderLine(models.Model):
     _inherit = "purchase.order.line"
 
-    @api.onchange("product_id")
-    def onchange_product_id(self):
-        # Change context in a compatible way with onchange usage
-        # See : https://github.com/odoo/odoo/issues/7472#issuecomment-119503916
-        self.env.context = frozendict(self.env.context, multiplier_qty_message=False)
-        res = super().onchange_product_id()
-        self.env.context = frozendict(self.env.context, multiplier_qty_message=True)
-        return res
-
     @api.onchange("product_qty")
-    def _onchange_quantity(self):
+    def onchange_product_qty_multiplier_qty(self):
```

Comment:
Please follow naming conventions:
```suggestion
    def _onchange_product_qty_multiplier_qty(self):
```

Reply:
Is it a convention ? 
I see both results in odoo. (Indeed more private function than public function)

- def onchange : 50 results in odoo/odoo/addons
- def _onchange : over 300 results in odoo/odoo/addons

nevermind, changed.

Reply:
Odoo doesn't serve for guidelines. Follow OCA guidelines:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#37models

Reply:
As corollary, I wanted to add some notes for this case where the onchange is not declared on the module that adds the field, and my take was to put `_onchange_<field_name>_<module_name>` as method name, for avoiding unintentional method overrides on 2 non-dependent modules, but I don't remember where this proposal goes (but definitively not merged).

Reply:
Hi @pedrobaeza. I applied your suggestion. (_onchange_product_qty_product_supplierinfo_qty_multiplier)
Is it OK for you ? thanks !